assets/src/edit-story/components/carousel/carouselList.js

```diff
@@ -49,22 +49,6 @@ const PageList = styled(Reorderable).attrs({
       overflow-x: visible;
       overflow-x: overlay;
       justify-content: flex-start;
-      /*
-       * These overrides are an exception - generally scrollbars should all
-       * look the same. We do this only here because this scrollbar is always visible
-       * if scroll is possible.
-       */
```

Comment:
Ok so please correct me here if my eyes are just playing tricks. I don't think that the override is getting to the scrollbar for the carousel. 
![Screen Shot 2021-03-01 at 11 03 51 AM](https://user-images.githubusercontent.com/10720454/109543130-32695380-7a83-11eb-87a4-e8cc94637efd.png)

I tweaked how the css is getting applied and retested and got this: 
![really tough to see horizontal scroll](https://user-images.githubusercontent.com/10720454/109543276-5167e580-7a83-11eb-8bbe-6985f56ad1d8.gif)

Which feels more in line with the colors. 

I'm not really sure the best way to handle this, perhaps an additional helper to override just colors?

Reply:
we can def add new colors, but the problem was that the scrollbar here wasn't being shown once the css wasn't being globally applied.

It's fixed now 
![small-scrollbar](https://user-images.githubusercontent.com/22185279/109550311-58472600-7a8c-11eb-9f4f-5cdc2067dc1e.gif)